### DISCUSSION TITLE
Why aren’t some rules included in the ESLint rules implementation progress issue?

### DISCUSSION THREAD

**Discussion by Orenbek** [oxc-project/oxc#7044] [Q&A]
I noticed that some of the typescript-eslint rules do not appear in [this issue](https://github.com/oxc-project/oxc/issues/2180). For instance, typescript-eslint/no-unnecessary-condition and typescript-eslint/no-unnecessary-boolean-literal-compare.

**Comment by Boshen** [comment]
> This rule requires [type information](https://typescript-eslint.io/getting-started/typed-linting) to run.

We excluded all the type are rules.
